### ISSUE TITLE
Carpet Fixes crashes the server

### ISSUE THREAD

**Issue by SergeyMC9730** [fxmorin/carpet-fixes#32]
Crash log:
```
[17:13:03] [main/INFO]: Loading Minecraft 1.18.1 with Fabric Loader 0.13.0
[17:13:04] [main/WARN]: Warnings were found!
 - Mod 'Fabric Rendering Fluids (v1)' (fabric-rendering-fluids-v1) 0.1.19+3ac43d9565 recommends any version of fabric-textures-v0, which is disabled for this environment (client/server only)!
	 - You should install any version of fabric-textures-v0 for the optimal experience.
[17:13:04] [main/INFO]: Loading 46 mods:
	- cardboard 49
	- carpet 1.4.57
	- carpet-fixes 1.18.1-1.5.1+v220107
	- com_github_isaiahpatton_specialsource master-SNAPSHOT via cardboard
	- config2brigadier 1.0.2 via fabrictailor
	- fabric 0.46.4+1.18
	- fabric-api-base 0.4.2+d7c144a865 via fabric
	- fabric-api-lookup-api-v1 1.5.3+d7c144a865 via fabric
	- fabric-biome-api-v1 6.0.2+d7c144a865 via fabric
	- fabric-command-api-v1 1.1.7+d7c144a865 via fabric
	- fabric-commands-v0 0.2.6+b4f4f6cd65 via fabric
	- fabric-containers-v0 0.1.19+d7c144a865 via fabric
	- fabric-content-registries-v0 0.4.8+d7c144a865 via fabric
	- fabric-crash-report-info-v1 0.1.9+3ac43d9565 via fabric
	- fabric-dimensions-v1 2.1.10+a1d9bbf565 via fabric
	- fabric-entity-events-v1 1.4.6+d7c144a865 via fabric
	- fabric-events-interaction-v0 0.4.17+d7c144a865 via fabric
	- fabric-events-lifecycle-v0 0.2.9+d7c144a865 via fabric
	- fabric-game-rule-api-v1 1.0.11+d7c144a865 via fabric
	- fabric-item-api-v1 1.3.1+691a79b565 via fabric
	- fabric-item-groups-v0 0.3.6+3ac43d9565 via fabric
	- fabric-lifecycle-events-v1 1.4.13+713c266865 via fabric
	- fabric-loot-tables-v1 1.0.9+d7c144a865 via fabric
	- fabric-mining-level-api-v1 1.0.6+d7c144a865 via fabric
	- fabric-mining-levels-v0 0.1.11+b4f4f6cd65 via fabric
	- fabric-networking-api-v1 1.0.19+d7c144a865 via fabric
	- fabric-networking-v0 0.3.6+b4f4f6cd65 via fabric
	- fabric-object-builder-api-v1 1.11.4+d7c144a865 via fabric
	- fabric-object-builders-v0 0.7.12+d7c144a865 via fabric
	- fabric-particles-v1 0.2.10+526dc1ac65 via fabric
	- fabric-permissions-api-v0 0.1-SNAPSHOT via fabrictailor
	- fabric-registry-sync-v0 0.9.2+ad01bfbd65 via fabric
	- fabric-rendering-data-attachment-v1 0.3.5+d7c144a865 via fabric
	- fabric-rendering-fluids-v1 0.1.19+3ac43d9565 via fabric
	- fabric-resource-conditions-api-v1 1.0.2+d7c144a865 via fabric
	- fabric-resource-loader-v0 0.4.14+713c266865 via fabric
	- fabric-screen-handler-api-v1 1.1.12+d7c144a865 via fabric
	- fabric-structure-api-v1 2.1.3+d7c144a865 via fabric
	- fabric-tag-extensions-v0 1.2.8+d7c144a865 via fabric
	- fabric-tool-attribute-api-v1 1.3.8+fb3b57b465 via fabric
	- fabric-transfer-api-v1 1.5.9+d7c144a865 via fabric
	- fabricloader 0.13.0
	- fabrictailor 1.8.0
	- icommon 1.18
	- java 17
	- minecraft 1.18.1
[17:13:04] [main/WARN]: Mod `com_github_isaiahpatton_specialsource` (master-SNAPSHOT) does not respect SemVer - comparison support is limited.
[17:13:04] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/home/container/libraries/net/fabricmc/sponge-mixin/0.11.1+mixin.0.8.5/sponge-mixin-0.11.1+mixin.0.8.5.jar Service=Knot/Fabric Env=SERVER
[17:13:04] [main/INFO]: Compatibility level set to JAVA_16
[17:13:04] [main/INFO]: Please wait, Scanning plugins for events...
[17:13:07] [main/INFO]: Found: 59 (Took: 3328ms)
[17:13:07] [main/INFO]: Loading Libraries...
[17:13:08] [main/INFO]: Debug: Loading library paper-api-1.17-dev.jar
[17:13:08] [main/INFO]: Debug: Loading library sqlite-jdbc-3.21.0.1.jar
[17:13:08] [main/INFO]: Debug: Loading library mysql-connector-java-5.1.46.jar
[17:13:08] [main/INFO]: Debug: Loading library commons-lang-2.6.jar
[17:13:08] [main/INFO]: Debug: Loading library commons-collections4-4.4.jar
[17:13:08] [main/INFO]: Debug: Loading library commons-collections-3.2.1.jar
[17:13:09] [main/INFO]: =======================================================
[17:13:09] [main/INFO]:  iCommon - Isaiah's common library for mods.
[17:13:09] [main/INFO]:  Copyright (c) 2018-2021 by Isaiah. Running on MC 1.18.1
[17:13:09] [main/INFO]: =======================================================
[17:13:09] [main/INFO]: Applying mixin: MixinMinecraftServer...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinEntity...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinPlayer...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinBlockEntity...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinChestBlockEntity...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinCampfireBlockEntity...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinScreenHandler...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinChunkGenerator...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinPersistentStateManager...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinPlayerManager...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinHeighttmap...
[17:13:10] [main/INFO]: Applying mixin: R1_18.MixinWorld_18...
[17:13:15] [main/WARN]: Method overwrite conflict for method_5800 in carpet-fixes.mixins.json:itemFixes.ItemEntity_lightningKillsDropsMixin, previously written by carpet.mixins.ItemEntityMixin. Skipping method.
[17:13:22] [main/WARN]: @Redirect conflict. Skipping carpet-fixes.mixins.json:conversionFixes.SlimeEntity_conversionMixin->@Redirect::ConversionFixSlime(Lnet/minecraft/class_1937;Lnet/minecraft/class_1297;)Z from mod carpet-fixes with priority 1000, already redirected by bukkitfabric.mixins.json:entity.MixinSlimeEntity->@Redirect::doBukkitEvent_RedirectSpawnEntity(Lnet/minecraft/class_1937;Lnet/minecraft/class_1297;)Z from mod cardboard with priority 1000
[17:13:22] [main/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1621 failed
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:418) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.13.0.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:66) [fabric-loader-0.13.0.jar:?]
	at net.fabricmc.installer.ServerLauncher.main(ServerLauncher.java:64) [fabric-server-mc.1.18.1-loader.0.13.0-launcher.0.10.2.jar:0.10.2]
Caused by: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1621 failed
	at net.minecraft.class_1299.<clinit>(class_1299.java:209) ~[server-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[server-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:81) ~[server-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:19) ~[server-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:20) ~[server-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:78) ~[server-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:77) ~[server-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:269) ~[server-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) ~[server-intermediary.jar:?]
	at net.minecraft.server.Main.main(Main.java:101) ~[server-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) ~[fabric-loader-0.13.0.jar:?]
	... 8 more
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1621 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:252) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.0.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1012) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:223) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:174) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.0.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:209) ~[server-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[server-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:81) ~[server-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:19) ~[server-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:20) ~[server-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:78) ~[server-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:77) ~[server-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:269) ~[server-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) ~[server-intermediary.jar:?]
	at net.minecraft.server.Main.main(Main.java:101) ~[server-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) ~[fabric-loader-0.13.0.jar:?]
	... 8 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.0.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1012) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:223) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:174) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.0.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:209) ~[server-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[server-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:81) ~[server-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:19) ~[server-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:20) ~[server-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:78) ~[server-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:77) ~[server-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:269) ~[server-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) ~[server-intermediary.jar:?]
	at net.minecraft.server.Main.main(Main.java:101) ~[server-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) ~[fabric-loader-0.13.0.jar:?]
	... 8 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Redirector ConversionFixSlime(Lnet/minecraft/class_1937;Lnet/minecraft/class_1297;)Z in carpet-fixes.mixins.json:conversionFixes.SlimeEntity_conversionMixin from mod carpet-fixes failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap carpet-fixes-refmap.json
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:468) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1381) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1052) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:400) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.11.1+mixin.0.8.5.jar:0.11.1+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.0.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1012) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:223) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:174) ~[fabric-loader-0.13.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.0.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:209) ~[server-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[server-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:81) ~[server-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:19) ~[server-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:20) ~[server-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:78) ~[server-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:77) ~[server-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:269) ~[server-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) ~[server-intermediary.jar:?]
	at net.minecraft.server.Main.main(Main.java:101) ~[server-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) ~[fabric-loader-0.13.0.jar:?]
	... 8 more
```
Used mods:

1. Cardboard 1.17-1.18 Latest
2. Carpet Fixes 1.18.1 15.1 v220107
3. Fabric API 0.46.4 1.18
4. Fabric Carpet 1.18.1 1.4.57 v220119
5. Fabrictailor 1.8.0
6. iCommon Fabric 1.18

**Comment by FxMorin** [comment]
Good to know that it's incompatible with cardboard, I'll attempt to fix it soon

**Comment by SergeyMC9730** [comment]
How do I compile this mod for 1.18.1? Gradlew compiles mod for 22w05a

**Comment by FxMorin** [comment]
oh right. Here: https://github.com/fxmorin/carpet-fixes/actions/runs/1815247245
Technically the 22w05a branch should work with 1.18.1 tho

**Comment by He3epM** [comment]
A potential solution has been determined, this may resolve your problem:
         - Replace mod 'ServerCore' (servercore) 1.5.1+1.20.1 with any version that is compatible with:
                 - cardboard 6
More details:
         - Mod 'ServerCore' (servercore) 1.5.1+1.20.1 is incompatible with any version of mod 'Cardboard' (cardboard), but a matching version is present: 6!
        at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:51) ~[fabric-loader-0.15.10.jar:?]
        at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:195) ~[fabric-loader-0.15.10.jar:?]
        at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:146) ~[fabric-loader-0.15.10.jar:?]
        at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68) ~[fabric-loader-0.15.10.jar:?]
        at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) ~[fabric-loader-0.15.10.jar:?]
        at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) ~[fabric-loader-0.15.10.jar:?]
        at net.fabricmc.installer.ServerLauncher.main(ServerLauncher.java:69) ~[server.jar:1.0.1]
        
        
please help me
